### ISSUE TITLE
POP STILL ON LrC v14.0 RELEASE

### ISSUE THREAD

**Issue by CATBirdlong** [ignaciocastro/a-dove-is-dumb#36]
### _非常抱歉, 该篇报告(正文)部分内容来自谷歌翻译, 原文为中文_

pop happened on _**Adove Ligjtroooom Classic v14.0 Release**_, 'About Ligtrooom'(Version) Screenshot-->![Screenshot 2024-12-15 012301](https://github.com/user-attachments/assets/c5883dbb-d312-4252-9ebe-32c2dda7e50f)

file 'hosts' already edited and the pop still
![Screenshot 2024-12-15 012217](https://github.com/user-attachments/assets/4e7f5826-ac55-4b0d-be40-6b1cbccb308b)
the file 'hosts' last updates is _# Last update: 2024-12-13 22:00 UTC_ -->[hosts.txt](https://github.com/user-attachments/files/18136981/hosts.txt)

but the rest of programs seem to be working :)

if you need the LrC  'Stand-alone Complete Installation Package' for such as test, 
Please contact '**_catbirdlong2926@foxmail.com_**(OR its Linked '**_44130139@qq.com_**')' 
SPARE:'**_whitecatbirdlong@gmail.com_**'
OR tg:'**_@CATBirdlong_**' and i already in your project's tg group.

### ANYWAY, you made something really different!_

**Comment by ignaciocastro** [comment]
早上好。此錯誤與子域名無關，因此您將無法獲得任何幫助，抱歉。請檢查您的安裝過程中出了什麼問題。

附註：如果您正在使用Clash，請注意，有一個專門的列表位於 https://a.dove.isdumb.one/clash

**Comment by CATBirdlong** [comment]
> 早上好。此錯誤與子域名無關，因此您將無法獲得任何幫助，抱歉。請檢查您的安裝過程中出了什麼問題。
> 
> 附註：如果您正在使用Clash，請注意，有一個專門的列表位於 https://a.dove.isdumb.one/clash

早上好, 感谢您对该issue的回复与处理.
我会尝试使用'文件检索工具'和'卸载工具'重新装载lrc.

顺带一提, 您在附注所提到的'如果您正在使用Clash...', 是否包括所有代理软件(如v2rayN), 我在昨天晚上才接触到您的工程, 所以对于您的工程如何实现效果完全一头雾水, 
如果您所说可能需要另一个适配列表专用于代理软件, 那么该pop很可能是代理软件造成的, 我并未操作代理软件方的黑名单, 我将在我的代理软件中配置它, 并且如果发生其他问题我可能将在该issue下继续回报信息.

总之, 非常感谢您的处理, 感谢您对该工程的付出!

_以下内容由Google Translate进行翻译并且已经过检查确保意思大多正确_
_The content has been translated by Google Translate and has been checked to ensure that the meaning is correctly_

Good morning, thank you for your reply to this issue.
I'll try to reinstall lrc using 'File Retrieval Tool' and 'Uninstall Tool'.

By the way, what you mentioned in the note 'If you're using Clash...', does it include all agent software (e.g. v2rayN), I only came across your project last night, so I'm completely confused about how your project will work out, 
If you say that another adaptation list may be needed specifically for the agent software, then the POP is most likely caused by the agent software, I am not operating the blacklist of the agent software side, I will configure it in my agent software, and if other problems occur I may continue to report information under this issue.

In short, thank you very much for your reply and thank you for your dedication to the project!
